plugins/reporters/opossum/src/main/kotlin/OpossumReporter.kt

```diff
@@ -500,53 +573,31 @@ class OpossumReporter(
         }
     }
 
+    @Serializable
+    internal data class OpossumInputMetadata(
+        val projectId: String = "0",
+        val fileCreationDate: String = LocalDateTime.now().toString()
+    )
+
     private fun writeReport(outputFile: File, opossumInput: OpossumInput) {
         val jsonFile = createOrtTempDir().resolve("input.json")
-        JsonMapper().setSerializationInclusion(Include.NON_NULL).writeValue(jsonFile, opossumInput.toJson())
+        val json = Json {
+            explicitNulls = false
+            encodeDefaults = true
+        }
```

Comment:
Currently, there is no need for this. I would rather not expose it thus if you are okay with that.

Reply:
Oh, I thought I saw somewhere that the test was creating its own `Json` instance which could have been avoided this way, but I do not see that code anymore. Maybe I was mistaken, or you changed in meanwhile. Anyway, let's keep it as-is then.

Reply:
There is a call to a `Json.parseToJsonElement` call in the functional test. May be you had that in mind? We could use the same JSON object there, but it is not necessary. But I can change that if you prefer?

Reply:
> But I can change that if you prefer?

Yes please. This ensure that if we ever change setting like `encodeDefaults` it's done consistently for all callers. In a way, the serializer belongs quite tightly to the data model itself, because the annotations to use in the data model depend on the serializer configuration.

Reply:
Done.